src/templates/header.hbs

```diff
@@ -75,61 +75,69 @@
         </a>
         <ul class="sub-nav-items">
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="1" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="1" class="nav-item sidebar-link"
               href="/hc/en-us/categories/1260801482329-minFraud-Web-Services">
               <span>See all guides</span>
             </a>
           </li>
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="2" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="2" class="nav-item sidebar-link"
               href='/hc/en-us/sections/4407832296475-Introduction-to-the-minFraud-Service'>
               <span>
                 Introduction to the minFraud Service
               </span>
             </a>
           </li>
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="3" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="3" class="nav-item sidebar-link"
               href='/hc/en-us/sections/4407465695259-Integration-and-Testing'>
               <span>
                 Integration and Testing
               </span>
             </a>
           </li>
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="4" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="4" class="nav-item sidebar-link"
               href='/hc/en-us/sections/4407474210459-Risk-Scoring'>
               <span>
                 Risk Scoring
               </span>
             </a>
           </li>
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="5" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="5" class="nav-item sidebar-link"
               href='/hc/en-us/sections/4407474216987-Risk-Data'>
               <span>
                 Risk Data
               </span>
             </a>
           </li>
           <li role="none" class="sidebar-item">
-            <a role="treeitem" aria-level="2" aria-setsize="9" aria-posinset="6" class="nav-item sidebar-link"
+            <a role="treeitem" aria-level="2" aria-setsize="10" aria-posinset="6" class="nav-item sidebar-link"
+              href='/hc/en-us/sections/5443666293147-Transaction-Data-Inputs'>
+              <span>
+                Transaction Inputs
```

Comment:
When the active section is `Transaction Data Inputs`, the `Transaction Inputs` sidebar nav item is not active (or highlighted) because of the difference in names. The highlighting of the active section in the sidebar nav item can be found in this [javascript function](https://github.com/maxmind-zendesk-theme/zendesk-theme/blob/main/src/scripts/ui/active-sidebar-link.js#L13-L19). You can probably do either of the following:
1. Make make the section name and the sidebar nav item name the same
2. Add a specific class to this nav item and function to the file above where if the current section is Transaction Data Inputs, highlight the Transaction Inputs section in the sidebar nav. This is similar to this previous (and now deleted) code [here](https://github.com/maxmind-zendesk-theme/zendesk-theme/pull/52/commits/29cfb4165eaef44ab15ea728ebd1f4aa3d4ed8e7#diff-aad4b0f9a6340de3d818632dea6533e421a3f23ffd060906964698c2866f15f1L8), that we had for the Correct GeoIP2 Data article. Correct GeoIP2 Data was not an actual section name, but we wanted it the sidebar nav item to be highlighted when a user was on it.

Reply:
i updated the name of the section on the Knowledge Base instead, because i prefer the title "Transaction Inputs" anyway